packages/compass-components/src/components/leafy-green/combobox/combobox.tsx

```diff
@@ -885,13 +886,18 @@ export default function Combobox<M extends boolean>({
       }
 
       setInputFocus(cursorPos);
+      // Open menu on click instead of on focus.
+      // https://jira.mongodb.org/browse/PD-2321
+      openMenu();
```

Comment:
When trying this combobox out I tried to click into the combobox here to open the dropdown to see the field names and it wasn't opening, it looks like the text input takes focus but the suggestions don't appear when nothing is typed. I think this ux was a bit confusing. Clicking on the dropdown arrow or on the border of the combobox, however, had the expected behavior. Should we update it to open on clicking the input?

I think a bigger case for accessibility is that now with a keyboard a user can't open the field name list without first typing part of the name. I'm thinking we either want to have the select field name menu open on enter or space like a select, or revert to the initial behavior with the menu opening on the wrapper focus.

Related jira ticket we were looking at https://jira.mongodb.org/browse/PD-2321

Reply:
This copied Commbobox is a temporary workaround until we can switch to the new design and the last version of LG components. Fixing all these accessibility issues for the old component would be tricky and time-consuming and considering that there is already [WIP PR](https://github.com/mongodb-js/compass/pull/3290) for using new styles we can keep it as is for a little longer while and then replace it with a new component where it all has ready fixed.

Reply:
Pushed an update though to open the menu on click on any input area.